docs/proposals/old-timer.md

```diff
@@ -0,0 +1,212 @@
+# Heapster Oldtimer
+
+## Overview
+
+Prior to the Heapster refactor, the Heapster model presented aggregations of
+metrics over certain time periods (the last hour and day).  Post-refactor, the
+concern of presenting an interface for historical metrics was to be split into
+a separate Heapster component: Oldtimer.
+
+Oldtimer will run as part of the main Heapster executable, and will present
```

Comment:
That would effectively mean that it would have to be on a different port.  Why not just allow it to run in the same executable, if you're going to be running it as part of the same pod anyway, so that `https://heapster.kube-system/api/v1/model` is for the model, while `https://heapster.kube-system/api/v1/historical` is for oldtimer?